src/sentry/integrations/bitbucket_server/client.py

```diff
@@ -139,24 +143,35 @@ def delete_hook(self, project, repo, webhook_id):
             auth=self.get_auth(),
         )
 
-    def get_commits(self, project, repo, fromHash, toHash):
-        return self.get(
+    def get_commits(self, project, repo, from_hash, to_hash, limit=1000):
+        logger.info(
+            "Loading commits for repo %s and project %s from %s to %s",
```

Comment:
@marbon87 It's better to put this information in the `extra` parameter of the logger so we can filter log messages easier on those parameters